app/enterprise/1.3-x/developer-portal/working-with-templates.md

```diff
@@ -302,3 +302,1181 @@ This is my second post!
 <h1>Post Two</h1>
 <p>This is my second post!</p>
 ```
+
+
+### List of Helper Functions
+
+# Kong Template Helpers - Lua API
+Kong Template Helpers are a collection of objects that not only give access to your portal data at time of render, but provide powerful integrations into Kong as well.
+
+
+## Globals
+
+- [`l`](#lkey-fallback) - Locale helper, first version, gets values from the currently active page.
+- [`each`](#eachlist_or_table) - Commonly used helper to iterate over lists / tables.
+- [`print`](#printany) - Commonly used helper to print lists / tables.
+- [`markdown`](#printany) - Commonly used helper to print lists / tables.
+- [`json_decode`](#json_decode) - Decode JSON to Lua table.
+- [`json_encode`](#json_encode) - Encode Lua table to JSON.
+
+
+## Objects
+
+- [`portal`](#portal) - The portal object refers to the current workspace portal being accessed.
+- [`page`](#page) - The page object refers to the currently active page, and it's contents.
+- [`user`](#user) - The user object represents the currently logged in developer accessing the Kong Portal.
+- [`theme`](#theme) - The theme object represents the currently active theme, and it's variables.
```

Comment:
```suggestion
- [`theme`](#theme) - The theme object represents the currently active theme and its variables.
```

Reply:
Again, Gayle's suggestions should be implemented.